roles/matrix-bridge-mautrix-signal/defaults/main.yml

```diff
@@ -93,6 +96,21 @@ matrix_mautrix_signal_configuration_extension_yaml: |
   #
   # If you need something more special, you can take full control by
   # completely redefining `matrix_mautrix_signal_configuration_yaml`.
+  #
+  # Permissions for using the bridge.
+  # Permitted values:
+  #      relay - Allowed to be relayed through the bridge, no access to commands.
+  #       user - Use the bridge with puppeting.
+  #      admin - Use and administrate the bridge.
+  # Permitted keys:
+  #        * - All Matrix users
+  #   domain - All users on that homeserver
+  #     mxid - Specific user
+  #
+  bridge:
+    permissions:
+      {{ matrix_mautrix_signal_homeserver_domain }}: "{{ "relay" if matrix_mautrix_signal_relaybot_enabled else "user" }}"
+
```

Comment:
I think relay and user are not exclusive.

I'd suggest moving this part back to config.yaml.j2 and change it (see below)

```suggestion
```